test/vm_tests/builder/parsing/members_test.dart

```diff
@@ -195,34 +195,31 @@ main() {
           };
 
           // Loop over the deprecated lifecycle methods
-          legacyLifecycleMethodsMap.keys.forEach((lifecycle) => {
```

Comment:
There were some syntax shenanigans going on here, it looks like from a TS/Dart arrow function syntax mixup 🙂.

This set of curly braces and the ones in the below for-loop are actually being interpreted as set literals, not blocks,  were causing warnings.

So, I fixed that and switched from forEach to a for-loop while I was at it.